### DISCUSSION TITLE
proposal for vacuum cleaner

### DISCUSSION THREAD

**Discussion by ChristophCaina** [home-assistant/frontend#14765] [Other feature requests]
Considering the following design sheet, there are many options to improve the colors for vacuum cleaner... :)

![grafik](https://user-images.githubusercontent.com/26391061/207476089-4eab750e-da28-4e3a-be4e-f2272e3686b9.png)
First of all, I would suggest to implement red for vacuum.error - to highlight the error state from the other icons.

On top of that, it could be discussed, if other colors should be used for the different states - or if pulsating icons could be used...
for example: I could imagine that "cleaning" could be a pulsating Icon, since the cleaner is performing a task - and is therefore in an active state.

btw: vacuum.cleaning does appear twice in this list... (?)

**Comment by ildar170975** [comment]
1. Each state mus have a corresponding theme variable

2. As for default colors:
![image](https://user-images.githubusercontent.com/71872483/207480645-d455c337-2e82-4216-93ea-3795939ba693.png)
and I do not now what are `on` & `off`.
